### PR TITLE
Added /give command!

### PR DESCRIPTION
**Pull request by ZNickq**
Works a bit like /i in some bukkit plugins:
/give znickq stone 10 -> gives znickq 10 stone!
/give stone 10 -> gives the player 10 stone!
/give stone -> gives the player 64 stone!

Only works for ops right now! :)

### CODE REVIEW

File: src/main/java/org/getspout/server/SpoutServer.java [hunk 1]
```diff
@@ -55,22 +55,7 @@
...
-import org.getspout.server.command.ListCommand;
-import org.getspout.server.command.MeCommand;
-import org.getspout.server.command.OpCommand;
-import org.getspout.server.command.ReloadCommand;
-import org.getspout.server.command.SaveCommand;
-import org.getspout.server.command.SayCommand;
-import org.getspout.server.command.StopCommand;
-import org.getspout.server.command.TimeCommand;
-import org.getspout.server.command.WhitelistCommand;
+import org.getspout.server.command.*;
```
Comment (Top-Cat) [inline src/main/java/org/getspout/server/SpoutServer.java]: Nope, nope, nope

Comment (ZNickq) [inline src/main/java/org/getspout/server/SpoutServer.java]: Fixing it now ;)
Netbeans does it automatically :/ Need to find a way to turn it off...

Comment (macalinao) [inline src/main/java/org/getspout/server/SpoutServer.java]: Doesn't do it for me.

File: src/main/java/org/getspout/server/SpoutServer.java [hunk 2]
```diff
@@ -460,6 +445,7 @@ public void start() {
 		commandMap.register(new SayCommand(this));
 		commandMap.removeAllOfType(ReloadCommand.class);
 		commandMap.register(new ReloadCommand(this));
+                commandMap.register(new GiveCommand(this));
```
Comment (Afforess) [inline src/main/java/org/getspout/server/SpoutServer.java]: Tabs vs. Spaces.

Comment (ZNickq) [inline src/main/java/org/getspout/server/SpoutServer.java]: Yeah, i fixed it in the next commit :)
Will work on fixing any other problems now!

File: src/main/java/org/getspout/server/SpoutServer.java (hunk 1 above)
Comment (ZNickq) [inline src/main/java/org/getspout/server/SpoutServer.java]: o_O when i add a certain number of imports, it just switches to *...wierd, i'll google it ;)

**Comment by ZNickq** [comment]
Ok, this should be it! I set netbeans to no longer do wildcards too! :)

**Comment by Afforess** [comment]
I think you should write this command so it can be executed from the console too.

**Comment by ZNickq** [comment]
@Afforess Ok, did that, also added /tp! I think my formatting is better now! :)

**Comment by Top-Cat** [comment]
Also you need a merge

**Comment by RoyAwesome** [comment]
Since we restructured Spout, this is irrelevant now.  Thanks though.

**Comment by ZNickq** [comment]
Np, but how did you restructure it? o-o

**Comment by RoyAwesome** [comment]
Blocks are going into Vanilla, so /give should probably live there (or take any registered block).  Not sure how to do that right now since blocks aren't in yet, but, we are getting there.

**Pull request closed without merging**